src/components/templates/Poster/components/IntroVideoPreviewModal/IntroVideoPreviewModal.scss

```diff
@@ -0,0 +1,30 @@
+@import "scss/constants";
+
+$iframe-width: 50vw;
+$iframe-height: calc(#{$iframe-width} * 9 / 16);
+$optimal-margin: 20px;
+
+.IntroVideoPreviewModal {
+  &__title {
+    font-size: 16px;
+    margin-bottom: $optimal-margin;
+    font-weight: 500;
+  }
+
+  &__iframe {
+    width: $iframe-width;
+    height: $iframe-height;
+    border: none;
+    border-radius: 15px;
```

Comment:
Can we extract this to a variable in `scss/constants` so we can start to get some uniformity across the codebase?

Reply:
extracted in https://github.com/sparkletown/sparkle/pull/1299/commits/fdbcb01ffff525018c80a932fbe38f2f5d1ddbe1
fixed in https://github.com/sparkletown/sparkle/pull/1299/commits/5ecef5628b53dc5f1587454b51b9ed7cd4195664